spec/features/trainees/submit_for_trn_spec.rb

```diff
@@ -2,130 +2,142 @@
 
 require "rails_helper"
 
-feature "submit for TRN" do
-  background { given_i_am_authenticated }
-
-  before do
-    stub_microsoft_oauth_success
-    stub_request(:post, "#{Settings.dttp.api_base_url}/contacts").to_return(
-      body: "",
-      headers: {
-        "OData-Version" => "4.0",
-        "OData-EntityId" => "https://dttp-dev.api.crm4.dynamics.com/api/data/v9.1/contacts(60f77a42-5f0e-e611-80e0-00155da84c03)",
-      },
-    )
-  end
+describe TraineeHelper do
```

Comment:
You should be able to include the `TraineeHelper` just once in this file:

```rb
feature "submit for TRN" do
  include TraineeHelper # new line

    ...

  describe "submission" do

    # your new test is in here somewhere, and should still have access to trainee_name

  end
end
```

Reply:
missed this 🤦🏼